sickbeard/search.py

```diff
@@ -160,6 +161,25 @@ def snatchEpisode(result, endStatus=SNATCHED):
 
         if curEpObj.status not in Quality.DOWNLOADED:
             notifiers.notify_snatch(curEpObj._format_pattern('%SN - %Sx%0E - %EN - %QN') + " from " + result.provider.name)
+            data  = {
+                    'seasons': [
+                        {
+                            'number': epObj.season,
+                            'episodes': [
+                                {
+                                    'number': epObj.episode
+                                }
+                            ]
+                        }
+                    ]
+
+             }
+
+            datas.update(data)
+
+    if sickbeard.USE_TRAKT:
```

Comment:
in the next commit is already changed. i have left USE_TRAKT becouse if you disable trakt notification but TRAKT_SYNC_WATCHLIST is enabled i think that it will update watchlist